app/queries/orders_query.rb

```diff
@@ -0,0 +1,17 @@
+class OrdersQuery
+  attr_accessor :initial_scope
+
+  def initialize(initial_scope = Orders.all)
+    @initial_scope = initial_scope
+  end
+
+  def call(params)
+    filter_by_status(initial_scope, params[:filter_by])
```

Comment:
initial_scope has an accessor, there is no need to pass it - I would also put params to accessors, so no arguments are passed at all
```suggestion
    filter_by_status(params[:filter_by])
```